lib/actions/ui.js

```diff
@@ -121,36 +123,36 @@ export function moveRight () {
   };
 }
 
-export function moveTo (i) {
+export function moveTo(i) {
   return (dispatch, getState) => {
     dispatch({
       type: UI_MOVE_TO,
       index: i,
-      effect () {
-        const { sessions } = getState();
+      effect() {
+        const {sessions} = getState();
         const uid = sessions.activeUid;
         const sessionUids = keys(sessions.sessions);
         if (uid === sessionUids[i]) {
           console.log('ignoring same uid');
-        } else if (null != sessionUids[i]) {
```

Comment:
Then I would rather throw a `TypeError` and be explicit about it.

But if you really want both `undefined` and `null`, I would strongly recommend just being verbose and explicit about it: `sessionUid[i] !== undefined && sessionUid[i] !== null`. Not everyone knows that `sessionUid[i] != null` coerces to the same. XO is about enforcing explicit and readable code.